driver-core/src/main/com/mongodb/internal/operation/ServerVersionHelper.java

```diff
@@ -18,35 +18,30 @@
 
 
 import com.mongodb.connection.ConnectionDescription;
-import com.mongodb.connection.ServerVersion;
 
 /**
  * This class is NOT part of the public API. It may change at any time without notification.
  */
 public final class ServerVersionHelper {
 
     public static boolean serverIsAtLeastVersionThreeDotZero(final ConnectionDescription description) {
```

Comment:
Could rename all these methods, e.g. 

    serverIsAtLeastWireVersionThree

or even inline them all, but that would require changing lots of operations.  I thought this was clearer.

Reply:
Agreed - you could rename them, but I like it as provides a way to determine the approx server version from the wire version.